lib/start.exs

```diff
@@ -1,3 +1,2 @@
-{:ok, pid} = Basic.Pipeline.start()
-Basic.Pipeline.play(pid)
+{:ok, _sup, _pipeline} = Basic.Pipeline.start_link()
```

Comment:
I you sure this will work? We have removed `start_link` injecting in `Membrane.Pipeline.__using__` macro, so I would say that it should be something more like: 
```
Membrane.Pipeline.start_link(Basic.Pipeline)
```